src/v/cluster/tx_gateway_frontend.cc

```diff
@@ -781,31 +540,16 @@ ss::future<try_abort_reply> tx_gateway_frontend::do_try_abort(
               pid,
               tx_seq);
             co_return try_abort_reply::make_aborted();
-        } else {
-            vlog(
-              txlog.warn,
-              "[tx_id={}] error looking up the transaction - {}",
-              tx_id,
-              tx_opt.error());
-            co_return try_abort_reply{tx::errc::unknown_server_error};
         }
+        vlog(
+          txlog.warn,
+          "[tx_id={}] error looking up the transaction - {}",
+          tx_id,
+          tx_opt.error());
+        co_return try_abort_reply{tx::errc::unknown_server_error};
     }
 
-    auto tx = tx_opt.value();
-
-    if (tx.transferring) {
-        tx_opt = co_await stm->reset_transferring(term, tx_id);
-        if (!tx_opt.has_value()) {
-            vlog(
-              txlog.warn,
-              "[tx_id={}] error rehydrating transaction - {}",
-              tx_id,
-              tx_opt.error());
-            // any error on lookin up a tx is a retriable error
-            co_return try_abort_reply{tx::errc::not_coordinator};
-        }
-        tx = tx_opt.value();
-    }
+    auto tx = std::move(tx_opt.value());
```

Comment:
Yes, that is true, i've only left the one that validates if the tranaction comes from future term, i do not think it can happen, will go over that once again